bot/handlers.py

```diff
@@ -515,6 +449,24 @@ def debt_sum(
                                            reply_markup=buttons.get_user_buttons(users))
         return States.DEBTOR_NAME
 
+    def cancel_button_handler(
+        self,
+        update: Update,
+        context: CallbackContext,
+    ):
+        data = update.callback_query.data
+        if data == menu_items.CANCEL:
```

Comment:
What will happen if I send "cancel" as a message?

Reply:
fallback handler will process your message and nothing will happen